examples/complete/compatability.tf

```
@@ -0,0 +1,282 @@
+  source  = "cloudposse/label/null"
+  version = "0.22.1"
+
+  # Known issue, additional_tag_map not taken from context
+  additional_tag_map = module.source_v25_22_full.context.additional_tag_map
+
+  context = module.source_v25_22_full.context
+}
+
+module "compat_25_24_full" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+
+  # Known issue, additional_tag_map not taken from context
+  additional_tag_map = module.source_v25_22_full.context.additional_tag_map
+
+  context = module.source_v25_24_full.context
+}
+
+module "compat_25_22_empty" {
+  source  = "cloudposse/label/null"
+  version = "0.22.1"
+
+  context = module.source_v25_empty.context
+}
+
+module "compat_25_24_empty" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+
+  context = module.source_v25_empty.context
+}
+
+module "compare_22_25_full" {
+  source = "./module/compare"
+  a      = module.source_v22_full
+  b      = module.compat_22_25_full
+}
+
+output "compare_22_25_full" {
+  value = module.compare_22_25_full
+}
+
+/* Uncomment this code to see how the fields differ
+output "source_22_full_id_full" {
+  value = module.source_v22_full.id_full
+}
+output "compat_22_25_full_id_full" {
+  value = module.compat_22_25_full.id_full
+}
+output "source_22_full_talm" {
+  value = module.source_v22_full.tags_as_list_of_maps
+}
+output "compat_22_25_full_talm" {
+  value = module.compat_22_25_full.tags_as_list_of_maps
+}
+*/
+
+module "compare_24_25_full" {
+  source = "./module/compare"
+  a      = module.source_v24_full
+  b      = module.compat_24_25_full
+}
+
+output "compare_24_25_full" {
+  value = module.compare_24_25_full
+}
+
+module "compare_22_25_empty" {
+  source = "./module/compare"
+  a      = module.source_v22_empty
+  b      = module.compat_22_25_empty
+}
+
+output "compare_22_25_empty" {
+  value = module.compare_22_25_empty
+}
+
+module "compare_24_25_empty" {
+  source = "./module/compare"
+  a      = module.source_v24_empty
+  b      = module.compat_24_25_empty
+}
+
+output "compare_24_25_empty" {
+  value = module.compare_24_25_empty
+}
+
+module "compare_25_22_full" {
+  source = "./module/compare"
+  a      = module.source_v25_22_full
+  b      = module.compat_25_22_full
+}
+
+output "compare_25_22_full" {
+  value = module.compare_25_22_full
+}
+
+module "compare_25_24_full" {
+  source = "./module/compare"
+  a      = module.source_v25_24_full
+  b      = module.compat_25_24_full
+}
+
+output "compare_25_24_full" {
+  value = module.compare_25_24_full
+}
+
+module "compare_25_22_empty" {
+  source = "./module/compare"
+  a      = module.source_v25_empty
+  b      = module.compat_25_22_empty
+}
+
+output "compare_25_22_empty" {
+  value = module.compare_25_22_empty
+}
+
+module "compare_25_24_empty" {
+  source = "./module/compare"
+  a      = module.source_v25_empty
+  b      = module.compat_25_24_empty
+}
+
+output "compare_25_24_empty" {
+  value = module.compare_25_24_empty
+}
+
+
+output "compatable" {
```

```suggestion
output "compatible" {
```

https://en.wiktionary.org/wiki/compatable